wcivf/apps/elections/tests/test_postcode_views.py

```diff
@@ -304,6 +304,54 @@ def test_multiple_elections_not_london(self, mock_response, client):
             response, "Polling stations are open from 8a.m. till 8p.m. today"
         )
 
+    def test_urls_in_list_of_ballots(self, mock_response, client):
+        # This test is to check that when there are two ballots of
+        # of different types on the same day, the correct url for the
+        # each ballot is included in the list of ballots
+        post_election1 = PostElectionFactory(
+            ballot_paper_id="local.city-of-london.aldgate.2021-05-06",
+            election__slug="local.city-of-london.2021-05-06",
+            election__election_date="2021-05-06",
+            election__election_type="local",
+        )
+        post_election2 = PostElectionFactory(
+            ballot_paper_id="parl.cities-of-london-and-westminster.2021-05-06",
+            election__slug="parl.2021-05-06",
+            election__election_date="2021-05-06",
+            election__election_type="parl",
+        )
+        # append the two post elections to the mock response
+        mock_response.json.return_value["dates"].extend(
+            [
+                {
+                    "date": post_election1.election.election_date,
+                    "polling_station": {"polling_station_known": False},
+                    "ballots": [
+                        {"ballot_paper_id": post_election1.ballot_paper_id}
+                    ],
+                },
+                {
+                    "date": post_election2.election.election_date,
+                    "polling_station": {"polling_station_known": False},
+                    "ballots": [
+                        {"ballot_paper_id": post_election2.ballot_paper_id}
+                    ],
+                },
+            ]
+        )
+
+        response = client.get(
+            reverse("postcode_view", kwargs={"postcode": "TE11ST"}), follow=True
+        )
+        asserts.assertContains(
+            response,
+            f'<a href="{post_election1.get_absolute_url()}">copeland</a>',
```

Comment:
I think it would be better to hard code these. At the moment if [get_absolute_url()](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/0918d80c363aedde14479b7c6f65134aaa38aee0/wcivf/apps/elections/models.py#L177) changes to return `None` (or whatever) then the test will pass but the link will be broken.